loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -243,7 +243,7 @@ static int binary_update_download_new_binary(void)
 	ret = write(write_fd, (FAR uint8_t *)&header_data, sizeof(binary_header_t));
 	if (ret != sizeof(binary_header_t)) {
 		printf("Failed to write header: %d\n", ret);
-		goto errout_with_close_fd1;
+		goto errout_with_close_fd2;
```

Comment:
In this case, `ret` can be greater than 0, but fail. But you return the ret. Is it okay?